tests/wrap.py

```diff
@@ -34,25 +34,22 @@ def testMakeControl(self):
         config.foo = 2
         config.bar.append("baz")
         control = config.makeControl()
-        self.assert_(testLib.checkControl(control, config.foo, config.bar))
+        self.assert_(testLib.checkControl(control, config.foo, config.bar._list))
```

Comment:
I also think that is nicer. But it requires upstream work. I'll discuss with John when we want to schedule that. It shouldn't be too much work, but more than the current ticket justifies IMHO.